modules/ROOT/pages/getting-started.adoc

```diff
@@ -48,28 +49,32 @@ qemu-kvm -m 2048 -cpu host -nographic -snapshot \
 qemu-img create -f qcow2 -b fedora-coreos-qemu.qcow2 my-fcos-vm.qcow2
 qemu-kvm -m 2048 -cpu host -nographic \
 	-drive if=virtio,file=my-fcos-vm.qcow2 \
-	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
+	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign \
+	-nic user,model=virtio,hostfwd=tcp::2222-:22
 ----
 +
 .Example launching FCOS with virt-install
 [source, bash]
 ----
 virt-install --connect qemu:///system -n fcos -r 2048 --os-variant=fedora31 --import \
 	--graphics=none --disk size=10,backing_store=/path/to/fedora-coreos-qemu.qcow2 \
-	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign"
+	--qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign" \
+	--qemu-commandline="-device virtio-net,netdev=user.0 -netdev user,id=user.0,hostfwd=tcp::2222-:22"
 ----
 
 NOTE: When using `virt-install`, you must specify absolute paths to the image and the Ignition file.
 
 TIP: If running with SELinux enabled, you may need to change the label of the Ignition file to allow access: `chcon -t svirt_home_t path/to/example.ign`
 
-Once the VM has finished booting, its IP addresses will appear on the serial console. You can then SSH into the FCOS VM as the `core` user:
+Once the VM has finished booting, its IP addresses will appear on the serial console. You can then SSH into the FCOS VM as the `core` user (replacing `localhost` and `2222` with the IP address and port of the VM, if different values from the suggested values mentioned earlier were chosen):
 
 [source, bash]
 ----
-ssh core@<ip address>
+ssh core@localhost -p 2222
```

Comment:
I'd keep this unchanged since it should work as is for libvirt, and for QEMU we'll have the notice from the previous paragraph.

Reply:
Better yet, move to a `${LOCALPORT}` variable which is `2222` on qemu and `22` on libvirt.

Reply:
:+1: to keeping this unchanged, and have the note above in https://github.com/coreos/fedora-coreos-docs/pull/86#discussion_r443684364. `${LOCALPORT}` sounds good, but  if we did this I think it'd look neater to have an `${IPADDRESS}` variable too (choosing localhost or the one assigned by libvirt), IMHO, to avoid a variable substitution in one part of the line but deleting/overwriting `<ip address>` in another. Since the user would need to enter the IP address manually anyway in the unchanged version or with a variable, and it wouldn't be too much more typing for `-p 2222`, I've left this as it is. Though really this is just minor preference, and I'd still be happy using the variables mentioned with a nudge (either both the IPADDRESS and LOCALPORT or just LOCALPORT).